README.md

```diff
@@ -11,30 +11,30 @@ The ElastiCache Cluster will be deployed into the `live-0` VPC. This is done thr
 module "example_team_ec_cluster" {
     source = "github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster"
 
-    team_name                 = "example-team"
-    ec_engine                 = "redis"
-    engine_version            = "4.0.10"
-    parameter_group_name      = "default.redis4.0"
-    node_type                 = "cache.m3.medium"
-    number_of_nodes           = 1
-    port                      = 6379
-    business-unit             = "example-bu"
-    application               = "exampleapp"
-    is-production             = "false"
-    environment-name          = "development"
-    infrastructure-support    = "example-team@digtal.justice.gov.uk"
+  cluster_name           = "cloud-platform-live-0"
+  cluster_state_bucket   = "moj-cp-k8s-investigation-platform-terraform"
```

Comment:
For `cluster_name` and `cluster_state_bucket` we should avoid providing the actual values here as they will be provided automatically by the pipeline. We don't want these to be supplied by the user.

However, I'm not entirely sure how to best present this here. Perhaps we can keep the comment from [here](https://github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster/blob/master/example/main.tf#L8-L14) but pass something like `cluster_name = ${var.cluster_name}` (and define the variables above) to demonstrate how it'll be used inside cloud-platform-environments? What do you think?

(and the same could apply to the main README I suppose, for consistency)